core/templates/dev/head/pages/exploration_player/ConversationSkinDirective.js

```diff
@@ -656,6 +656,8 @@ oppia.directive('conversationSkin', [
                                   true;
                               }
                             );
+                          ExplorationPlayerService.recordLeaveForRefresherExp(
```

Comment:
@aks681 found a bug here -- this is in the wrong place. Even if he clicks No in the redirect modal, it's getting recorded.

Reply:
Thanks; fixed.